tests/integration/shell/test_enabled.py

```diff
@@ -20,7 +20,7 @@ class EnabledTest(ModuleCase):
     validate the use of shell processing for cmd.run on the salt command line
     and in templating
     '''
-    cmd = ("printf '%s\n' first second third | wc -l ; "
+    cmd = ("echo -e 'first\\nsecond\\nthird' | wc -l ; "
```

Comment:
Should it be `\\n` as you have it here or `\n`?

Reply:
It should remain as is since echo is being used instead of printf and they handle EOL differently